requirements-excluded.txt

```diff
@@ -0,0 +1,5 @@
+# Remove excluded libs, if installed - see https://github.com/MiczFlor/RPi-Jukebox-RFID/pull/2470
+# Libraries which must be excluded. These must be removed with
+# You need to uninstall these with `python -m pip uninstall -y -r requirements.txt`
```

Comment:
Good documentation of purpose, but can we add more details about why RPi.GPIO needs to be excluded.